src/components/CurrentWeatherData/index.js

```diff
@@ -1,20 +1,244 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useEffect, useState } from 'react';
+// Libs
+import { useDispatch, useSelector } from 'react-redux';
+import { Trans, useTranslation } from 'react-i18next';
+import { formatDistanceToNow } from 'date-fns';
+// Constants
+import { getWeatherIcon } from '../../constants/iconMapping';
+import { langMap } from '../../constants/language';
+// Redux
+import { fetchWeather, setLastUpdate } from '../../store/slices/weatherSlice';
+// Icons
+import { MdRefresh } from 'react-icons/md';
+// SVG
+import { Wave1, Wave2 } from './Waves';
+// Styles
+import styled from 'styled-components';
```

Comment:
In big projects the order of imports is usually:
1) external libs
2) internal libs
3) absolute paths (ie. webpack alias)
4) relative paths